### PR TITLE
Tribe owns buildings game mode

### PR DESCRIPTION
**Pull request by theQuazz**
- allocates a player per tribe to own buildings (from the 12 unused players)
- that player gets a random cool name like "Darkspear Tribe"
- that player automatically turns on shared control to other players on the tribe
- the mode also turns on ac (but not sc) for all players in the tribe (no more need to -ac)

### CODE REVIEW

**Comment by Jaccouille** [comment]
What's this for? no description >:(

**Comment by theQuazz** [comment]
@Jaccouille updated

File: wurst/systems/core/Tribe.wurst [hunk 1]
```diff
@@ -5,6 +5,30 @@ import TribeUtils
...
 import LegacyColors
+import GameConfig
+import GameMode
+
+let listeners = CreateTrigger()
+
+public function registerTribeInitializationFinishEvent(code listener)
+    listeners.addAction(listener)
+
+let names = asList(
```
Comment (jonathanmclaus) [inline wurst/systems/core/Tribe.wurst]: https://wowwiki.fandom.com/wiki/Troll_tribes

Comment (theQuazz) [reply wurst/systems/core/Tribe.wurst]: I saw that. I didn't want to go full WoW. Besides, I like mine more.

**Comment by theQuazz** [comment]
@jonathanmclaus @Jaccouille could use another approval

**Pull request merged**